File: APIRollTheDice-Backend/src/main/java/com/example/APIRollTheDice/APIRollTheDice_Backend/Model/DTO/GameDTO/LootTableDTO/LootTableDTO.java
package com.example.APIRollTheDice.APIRollTheDice_Backend.Model.DTO.GameDTO.LootTableDTO;

import com.example.APIRollTheDice.APIRollTheDice_Backend.Model.Obj.Game.LootTable.LootElement;

import java.util.List;

public class LootTableDTO {
    private Long id;
    private String name;
    private List<LootElementDTO> lootElements;
    private Long idGameBundle;

    public LootTableDTO(Long id, String name, List<LootElementDTO> lootElements, Long idGameBundle) {
        this.id = id;
        this.name = name;
        this.lootElements = lootElements;
        this.idGameBundle=idGameBundle;
    }

    public LootTableDTO() {
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<LootElementDTO> getLootElements() {
        return lootElements;
    }

    public void setLootElements(List<LootElementDTO> lootElements) {
        this.lootElements = lootElements;
    }

    public Long getIdGameBundle() {
        return idGameBundle;
    }

    public void setIdGameBundle(Long idGameBundle) {
        this.idGameBundle = idGameBundle;
    }
}

File: RollTheDice/Assets/_Project/API/Model/DTO/GameDTO/LootTableDTO/LootTableDTO.cs
using System.Collections.Generic;

namespace Assets._Project.API.Model.DTO.GameDTO.LootTableDTO
{
     public class LootTableDTO
    {
        
        public long Id { get; set; }
        public string Name { get; set; }
        public List<LootElementDTO> LootElements { get; set; }
        public long IdGameBundle { get; set; }

        public LootTableDTO(){}
        public LootTableDTO(long id, string name, long idGameBundle, List<LootElementDTO> lootElements)
        {
            Id = id;
            Name = name;
            IdGameBundle = idGameBundle;
            LootElements = lootElements;
        }
    }
}